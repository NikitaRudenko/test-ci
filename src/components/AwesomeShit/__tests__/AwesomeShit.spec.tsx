import * as React from 'react';
import {mount} from 'enzyme';

import {AwesomeShit} from '../';

describe('AwesomeShit', () => {
	it('Should render component', () => {
		const component = mount(<AwesomeShit />);
		expect(component).toMatchSnapshot();
	});
});
