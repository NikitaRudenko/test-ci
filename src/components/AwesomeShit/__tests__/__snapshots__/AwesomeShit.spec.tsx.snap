// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AwesomeShit Should render component 1`] = `
ReactWrapper {
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {},
    "ref": null,
    "rendered": Object {
      "instance": <div>
        Stupid component
      </div>,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "Stupid component",
      },
      "ref": null,
      "rendered": Array [
        "Stupid component",
      ],
      "type": "div",
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {},
      "ref": null,
      "rendered": Object {
        "instance": <div>
          Stupid component
        </div>,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Stupid component",
        },
        "ref": null,
        "rendered": Array [
          "Stupid component",
        ],
        "type": "div",
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
